packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,121 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationToast
+
+<Status.Stable />
+
+<Story id="notification-notificationtoast--base" />
+<Props />
+
+The notification toast component non-disruptively provides quick and contextual feedback to the user. It can be based on the outcome of an action (i.e. saving a form) or it can pop up without having been triggered explicitly (i.e. a toast message "You made a new sale" showing real time sales in an app).
+
+## When to use it
+
+The notification toast should be used for a feedback on micro actions (i.e. successfully/unsuccessfully saving some information).
```

Comment:
I'm not sure that this section is very useful (duplicates the intro), but let's first come up with a consistent docs structure and clean this up later if necessary

Reply:
I removed this part in the end.